content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,50 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Delegation of managing a Job object to external controller
+
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
+
+{{< note >}}
+You can only set the `managedBy` field on Jobs if you enable the `JobManagedBy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+(disabled by default).
+{{< /note >}}
+
+This feature allows you to disable the built-in Job controller, for a specific
+Job, and delegate reconciliation of the Job to an external controller.
+
+You indicate the controller that reconciles the Job by setting a custom value
+for the `spec.managedBy` field - any value
+other than `kubernetes.io/job-controller`. The value of the field is immutable.
+
+{{< note >}}
+When using this feature, make sure the controller indicated by the field is
+installed, otherwise the Job may not be reconciled at all.
+{{< /note >}}
+
+{{< note >}}
+When developing an external Job controller be aware that your controller needs
+to operate in a fashion conformant with the definitions of the API spec and
+status fields of the Job object.
+
+Please review these in detail the [Job API](/docs/reference/kubernetes-api/workload-resources/job-v1/).
+We also recommend you running the e2e conformance tests for the Job object to
+verify your implementation.
```

Comment:
```suggestion
Please review these in detail in the [Job API](/docs/reference/kubernetes-api/workload-resources/job-v1/).
We also recommend that you run the e2e conformance tests for the Job object to
verify your implementation.
```

Reply:
Sorry, I was too slow to handle that. @atiratree do you think these are important enough for follow up PR, or just let it stay as is?

Reply:
@mimowo  I would prefer an update if you agree that these formulations sound a bit off. I am not saying my version is the best though, so I will leave the final version up to you..

Reply:
I think both suggestions are slightly better than original form so let me open the PR, we will see if other folks have other proposals there

Reply:
Follow up: https://github.com/kubernetes/website/pull/45506